src/styles/base.player.styl

```
@@ -62,6 +62,27 @@
 			left 50%
 			top 50%
 
+		.static
+			filter grayscale(60%)
+			background #222
+			position absolute
```

The box layer is too big – far extends the actual player size.

```suggestion
			position absolute
			bottom 0
			right 0
			left 0
```